var/spack/repos/builtin.mock/packages/xz/package.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+class Xz(AutotoolsPackage):
```

Comment:
Same here, with `libtool-dependency`. Actually, maybe the whole thing should be changed to `version-test-dependent ^version-test-pkg ^version-test-dependency-preferred`